integration-tests/python/default_instrumentation_python_env_variables.json

```diff
@@ -0,0 +1,13 @@
+
+{
+  "PYTHONPATH":                              "/otel-auto-instrumentation-python/opentelemetry/instrumentation/auto_instrumentation:/otel-auto-instrumentation-python",
```

Comment:
these values are incorrect, it should be the env vars defined here: https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/pkg/instrumentation/defaultinstrumentation.go#L91

I think these are the namespaced ones, not the default instrumentation vars